content/docs/command-reference/data/status.md

```diff
@@ -43,32 +43,37 @@ DVC uncommitted changes:
 (there are other changes not tracked by dvc, use "git status" to see)
 ```
 
-As shown above, the `dvc data status` displays changes in multiple categories:
-
-- _Not in cache_ indicates that the hash for files are recorded in `dvc.lock`
-  and `.dvc` files but the corresponding cache files are missing.
-- _DVC committed changes_ indicates that there are changes that are
-  `dvc-commit`-ed that differs with the last Git commit. There might be more
-  detailed state on how each of those files changed: _added_, _modified_,
-  _deleted_ and _unknown_.
-- _DVC uncommitted changes_ indicates that there are changes in the working
-  directory that are not `dvc commit`-ed yet. Same as _DVC committed changes_,
-  there might be more detailed state on how each of those files changed.
-- _Untracked files_ shows the files that are not being tracked by DVC and Git.
-  This is disabled by default, unless [`--untracked-files`](#--untracked-files)
-  is specified.
-- _DVC unchanged files_ shows the files that are not changed. This is not shown
-  by default, unless [`--unchanged`](#--unchanged) is specified.
-
-By default, `dvc data status` does not show individual changes inside the
-tracked directories, which can be enabled with [`--granular`](#--granular)
-option.
+`dvc data status` displays changes in multiple categories:
+
+- _Not in cache_ indicates that there are file records (hashes) in `.dvc` or
```

Comment:
Let's not introduce the term `file records`, it's not the same thing as "file hashes are recorded". The `status` says "file hash(es)".

From `dvc status` docs:
> not in cache: An output exists in the workspace, and the corresponding file hash in the [dvc.lock](https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#dvclock-file) or [.dvc](https://dvc.org/doc/user-guide/project-structure/dvc-files) file is up to date, but there is no corresponding cache file or directory.

Reply:
Changed to "file hashes".